app/hooks/hooks.types.ts

```diff
@@ -8,6 +9,45 @@ type mobileVersions = {
   platform: string
 }
 
+type Wallet = {
```

Comment:
type should be written manually. they should be imported from codegen 

ie: https://github.com/GaloyMoney/galoy-client/blob/36929772cec9c12c796a37b07b6c3dc61c2dfca9/package.json#L11